### DISCUSSION TITLE
在哪里可以找到relax模块下的接口文档？

### DISCUSSION THREAD

**Discussion by un-knight** [mlc-ai/mlc-zh#116] [Q&A] [answered]
在TVM官方文档（[https://chinese.tvm.wiki/index.html](https://chinese.tvm.wiki/index.html)）中没找到relax模块下的各种接口文档信息，比如`BlockBuilder`，`emit`等接口。不知道在哪里可以查看到这些接口信息？

看到 #44 猜测是目前relax还没合并到TVM main中所以文档没有相应信息，不过如果在这个过渡期想了解relax接口文档，不知道是否有地方查询？还是需要看源码？

**Comment by Hzfengsy** [comment, marked as answer]
感谢提问，说明几点：
1. https://chinese.tvm.wiki/index.html 不是TVM官方文档，是国内社区自发构建的一个翻译版本，目前已经较长时间没有更新。官方文档请参阅 https://tvm.apache.org/docs/
2. relax并未合并TVM main，目前文档也相对较少。可以考虑查看https://github.com/tlc-pack/relax/wiki ，或者查阅源码
